### PR TITLE
correction and add fakeprinter script

### PR DESCRIPTION
**Pull request by patrickelectric**

### CODE REVIEW

**Review by sithlord48 — APPROVED** [review]

File: scripts/fakeprinter.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/python
+
+import sys
+import time
+import serial
+
+fwlist = {
+	'repetier' : """"FIRMWARE_NAME: Repetier_0.92.9
```
Comment (sithlord48) [inline scripts/fakeprinter.py]: """" to open and """ to close ? is there one extra opening " ?

File: scripts/fakeprinter.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/python
+
+import sys
+import time
+import serial
+
+fwlist = {
```
Comment (sithlord48) [inline scripts/fakeprinter.py:7]: someother firmware strings that can be added

'aprinter' : """ ok FIRMWARE_NAME:APrinter"""

'teacup' :  """FIRMWARE_NAME:Teacup FIRMWARE_URL:http://github.com/traumflug/Teacup_Firmware/ PROTOCOL_VERSION:1.0 MACHINE_TYPE:Mendel EXTRUDER_COUNT:1 TEMP_SENSOR_COUNT:2 HEATER_COUNT:3"""
(other marlin was a custom version) 

'marlin' : """FIRMWARE_NAME:Marlin 1.1.0-RC7 (Github) SOURCE_CODE_URL:https://github.com/MarlinFirmware/Marlin PROTOCOL_VERSION:1.0 MACHINE_TYPE:3D Printer EXTRUDER_COUNT:1 UUID:cede2a2f-41a2-4748-9b12-c55c62f367ff"""

File: scripts/fakeprinter.py (hunk 1 above)
Comment (patrickelectric) [reply scripts/fakeprinter.py]: Ok

File: scripts/fakeprinter.py (hunk 2 above)
Comment (patrickelectric) [reply scripts/fakeprinter.py:7]: About that: Marlin, Teacup, APrinter do not have spaces ?

**Pull request merged**